src/extractScripts.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+const fs = require('fs');
+const htmlparser2 = require('htmlparser2');
+const tmp = require('tmp');
+
+const SCRIPT = 'script';
+// IMPORTANT: if file naming convention changes,
+// update corresponding strings in testExtractScripts.js
+const TEMP_DIR = 'tempJsScripts';
+
+/**
+ * extractScriptsFromHTML extracts the javascript from within <script>
+ * tags in the HTML test provided and writes each script to a .js file.
+ * It creates a directory "tempJsScripts-{tmp numbers}" in the current
```

Comment:
Sounds like this should be in both `.gitignore` and `.eslintignore`.

Reply:
_In theory_ the directory is cleaned up always, but yes you're right, thanks!